formats/finder_email_signup/publisher/links.json

```diff
@@ -0,0 +1,169 @@
+{
```

Comment:
This schema doesn't seem right at all. We should only be passing content_ids, not details - those are expanded by publishing-api itself. And it looks like the only keys actually needed in the links are oragnisations and related, both of which are just refs to guid_list which is already defined.